library/src/main/kotlin/com/a99/rxplaces/RxTextView.kt

```diff
@@ -0,0 +1,54 @@
+package com.a99.rxplaces
+
+import android.text.Editable
+import android.text.TextWatcher
+import android.widget.TextView
+import rx.Observable
+import rx.Subscriber
+import rx.android.MainThreadSubscription
+import rx.android.MainThreadSubscription.verifyMainThread
+
+class RxTextView {
+
+  companion object {
+    fun textChanges(textView: TextView): Observable<CharSequence> {
+      val observable = Observable.create<CharSequence> { subscriber ->
```

Comment:
I've tried so hard to avoid this creator but I couldn't. I believe that we could create an issue to improve this along with an issue to migrate the project to RxJava 2.